[Bug]: ImportError: cannot import name 'EpisodeType' from 'memmachine.common.api'

### Describe the bug

In the latest dev build, the MemMachine-app fails to start with this error displayed in the logs

```
memmachine-app       | Traceback (most recent call last):
memmachine-app       |   File "/app/.venv/bin/memmachine-server", line 4, in <module>
memmachine-app       |     from memmachine.server.app import main
memmachine-app       |   File "/app/.venv/lib/python3.12/site-packages/memmachine/__init__.py", line 4, in <module>
memmachine-app       |     from memmachine.rest_client import MemMachineClient, Memory, Project
memmachine-app       |   File "/app/.venv/lib/python3.12/site-packages/memmachine/rest_client/__init__.py", line 8, in <module>
memmachine-app       |     from .client import MemMachineClient
memmachine-app       |   File "/app/.venv/lib/python3.12/site-packages/memmachine/rest_client/client.py", line 16, in <module>
memmachine-app       |     from memmachine.common.api.spec import (
memmachine-app       |   File "/app/.venv/lib/python3.12/site-packages/memmachine/common/api/spec.py", line 10, in <module>
memmachine-app       |     from memmachine.common.api import EpisodeType, MemoryType
memmachine-app       | ImportError: cannot import name 'EpisodeType' from 'memmachine.common.api' (/app/.venv/lib/python3.12/site-packages/memmachine/common/api/__init__.py)
```

### Steps to reproduce

1. Build a new Docker image locally using `build-docker.sh`
2. Verify the new Docker image built successfully
3. run `memamchine-compose.sh start memmachine/memmachine:<tag>`
4. Wait for memmachine to fail to start
5. Check the logs using `memmachine-compose.sh logs`

eg:

Here is the locally created Docker image built using `build-docker.sh`

```
$ docker image ls
REPOSITORY              TAG                        IMAGE ID       CREATED             SIZE
memmachine/memmachine   0.2.3.dev9_g4ae4c39a-cpu   ddcc77cad20e   About an hour ago   459MB
```

Start memmachine and specify the new Docker image:

```
$ ./memmachine-compose.sh start memmachine/memmachine:0.2.3.dev9_g4ae4c39a-cpu
MemMachine Docker Startup Script
====================================

[SUCCESS] Docker and Docker Compose are available
[SUCCESS] .env file found
[SUCCESS] configuration.yml file found
[SUCCESS] OpenAI API key appears to be configured
[SUCCESS] OPENAI_API_KEY is configured
[SUCCESS] API key in configuration.yml appears to be configured
[SUCCESS] Database credentials in configuration.yml appear to be configured
[INFO] Pulling and starting MemMachine services...
[INFO] Pulling latest images... (Target: memmachine/memmachine:0.2.3.dev9_g4ae4c39a-cpu)
[WARNING] Image 'memmachine/memmachine:0.2.3.dev9_g4ae4c39a-cpu' not found in Docker Hub registry (manifest unknown). Assuming local image.
[+] Running 4/4
 ✔ Network memmachine-network     Created                                                                                                                                                                                                                                                                         0.0s 
 ✔ Container memmachine-neo4j     Healthy                                                                                                                                                                                                                                                                        34.7s 
 ✔ Container memmachine-postgres  Healthy                                                                                                                                                                                                                                                                         5.7s 
 ✔ Container memmachine-app       Started                                                                                                                                                                                                                                                                        34.8s 
[SUCCESS] Services started successfully!
[INFO] Waiting for services to be healthy...
NAME                  IMAGE                                            COMMAND                  SERVICE      CREATED          STATUS                                     PORTS
memmachine-app        memmachine/memmachine:0.2.3.dev9_g4ae4c39a-cpu   "sh -c memmachine-se…"   memmachine   35 seconds ago   Up Less than a second (health: starting)   0.0.0.0:8080->8080/tcp, [::]:8080->8080/tcp
memmachine-neo4j      neo4j:5.23-community                             "tini -g -- /startup…"   neo4j        35 seconds ago   Up 34 seconds (healthy)                    0.0.0.0:7473-7474->7473-7474/tcp, [::]:7473-7474->7473-7474/tcp, 0.0.0.0:7687->7687/tcp, [::]:7687->7687/tcp
memmachine-postgres   pgvector/pgvector:pg16                           "docker-entrypoint.s…"   postgres     35 seconds ago   Up 34 seconds (healthy)                    0.0.0.0:5432->5432/tcp, [::]:5432->5432/tcp
[INFO] Checking service health...
[INFO] Waiting for PostgreSQL to be ready...
/var/run/postgresql:5432 - accepting connections
[SUCCESS] PostgreSQL is ready
[INFO] Waiting for Neo4j to be ready...
[SUCCESS] Neo4j is ready
[INFO] Waiting for MemMachine to be ready...
[ERROR] MemMachine failed to become ready in 120 seconds. Check container logs and configuration.
```



### Expected behavior

`memmachine-app` should start without reporting any errors.

### Environment

OS: Ubuntu Linux 25.04
MemMachine: v0.2.3.dev9_g4ae4c39a-cpu

### Additional context

_No response_

--- a/src/memmachine/common/api/spec.py
+++ b/src/memmachine/common/api/spec.py
@@ -7,8 +7,8 @@ from typing import Annotated, Any, Self
 import regex
 from pydantic import AfterValidator, BaseModel, Field, field_validator, model_validator
 
-from memmachine.common.api import EpisodeType, MemoryType
-from memmachine.common.api.doc import Examples, SpecDoc
+from . import EpisodeType, MemoryType
+from .doc import Examples, SpecDoc
 
 DEFAULT_ORG_AND_PROJECT_ID = "universal"
 

